packages/mgt-components/src/components/mgt-people-picker/strings.ts

```diff
@@ -9,10 +9,9 @@ export const strings = {
   inputPlaceholderText: 'Start typing a name',
   noResultsFound: "We didn't find any matches.",
   loadingMessage: 'Loading...',
-  suggestedContact: 'suggested contact',
-  suggestedContacts: 'suggested contacts',
   selected: 'selected',
   removeSelectedItem: 'remove selected item',
   removeSelectedUser: 'Remove ',
-  selectContact: 'select a contact'
+  selectContact: 'select a contact',
+  suggestionsTitle: 'Suggested Contacts'
```

Comment:
We will need to update both the localization story and the docs for the people picker

Reply:
Good catch, thanks!

Reply:
PR to your PR for the docs update: https://github.com/microsoftgraph/microsoft-graph-docs/pull/21788